### ISSUE TITLE
Brikkit Crash

### ISSUE THREAD

**Issue by n42k** [n42k/brikkit#5]
Supposedly a random crash.

Stack trace:
```
--- SERVER END ---
Error: Invalid chat event: sender not a player.
    at new ChatEvent (/snapshot/brikkit/events/chat.js:0:0)
    at Brikkit._handleBrickadiaLine (/snapshot/brikkit/brikkit.js:0:0)
    at constructor._brickadia.on.line (/snapshot/brikkit/brikkit.js:0:0)
    at Interface.constructor.outRl.on.line (/snapshot/brikkit/brickadia.js:0:0)
    at emitOne (events.js:116:13)
    at Interface.emit (events.js:211:7)
    at Interface._onLine (readline.js:280:10)
    at Interface._normalWrite (readline.js:422:12)
    at Socket.ondata (readline.js:139:10)
    at emitOne (events.js:116:13)
```

**Comment by n42k** [comment]
This crash, and many similar ones, are related to players joining. `sm_join.js` will have to be fixed/rewritten.

**Comment by n42k** [comment]
Closed by 5e9fd866e6a6e06c0ad4731eccf02ea3705b819c.
